### ISSUE TITLE
Login page and other pages does not show up properly

### ISSUE THREAD

**Issue by mervynteo** [nolimits4web/Framework7#1076]
This is a (multiple allowed):
- [x] bug
- [ ] enhancement
- [ ] feature-discussion (RFC)
- Framework7 Version: Latest
- Platform and Target: IOS and Android

**Attached is a github repo for the problem. You require Meteor.** 
[**# HERES**](https://github.com/mervynteo/problemdemo) the link 
### What you did

**2 problems occur when I switch over the router.** 
problem 1. I have a login / register page after routing from home page. Issues occur on both iron-router and flow-router.
problem 2. When I enter the app, the tabs bar (toolbar) is directly under the navbar. Sometimes it overlaps the covers the navbar. Layout is set in layout.html. Within each html template, I define both navbar and contents. It seems like page-content is unable to load into "main" within layout.html. Typing in some random text shows up in between the navbar and footer tabbar.
### Expected Behaviour

I expect to see something but currently is blank. However, if I remove the first 3 divs (login screen, view and class) of the html,the screen appears but the formatting is not right. 

**problem 1 - Error with login/register page**
<img width="389" alt="error screenshot" src="https://cloud.githubusercontent.com/assets/5101947/17219403/d73c0a6e-551d-11e6-956e-26ba90f4e6ad.png">

**problem 2 - main content page doesnt load, footer bar floats to top**
<img width="369" alt="problem sreen" src="https://cloud.githubusercontent.com/assets/5101947/17274807/cdef75c8-5724-11e6-96f5-ab909bca4042.png">
### Actual Behavior

From the home landing page, clicking the login or register button, the login page does not load up. However the home page works well.
1. After removing 3 divs (login-screen, view and page), shows a barebone login with bad formatting
2. For the other pages, I have to remove 2 divs from layout (views and view-main), in order for the page to show the wrong format navbar and footer tabbar just below it.

**Comment by ZanderBrown** [comment]
If you want the login screen visible when you open the app you need to add the 'modal-in' class to your login-screen div

**Comment by mervynteo** [comment]
I tried that, didn't work as well.

**Comment by ZanderBrown** [comment]
Do you can a initialised app as well? Could you share more of your code?

**Comment by ZanderBrown** [comment]
What do you mean 'as well' your issue suggests nothing you had tried had worked at all

**Comment by mervynteo** [comment]
@ZanderBrown Thanks for replying. I have it on local development. Which parts would be relevant to you? I tried to troubleshoot by inserting helpers, no error messages or signals at all. Nothing on console, the routes work ok and log as it should. 

I did notice when I tried to edit a conditional helper. So if its a specific route name, hide the footer. Initially it worked in iron router but when I swapped over to flow router, I have the error above.
